sessionize/lib/src/iosMain/kotlin/co/touchlab/sessionize/platform/Notifications.kt

```diff
@@ -25,7 +25,7 @@ import platform.darwin.NSObject
 private val localNotificationDelegate = LocalNotificationDelegate()
 
 @ExperimentalUnsignedTypes
-actual fun createLocalNotification(title:String, message:String, timeInMS:Long, notificationId: Int) {
+actual fun createLocalNotificationOnPlatform(title:String, message:String, timeInMS:Long, notificationId: Int) {
 
     val date = NSDate.dateWithTimeIntervalSince1970(timeInMS / 1000.0)
```

Comment:
Per our discussion. Probably don't want to do this in interop

Reply:
Moved the notification code outside of the interop into the platform level. Created new class files for NotificationApiImpl